docs/reference/overview/excel-add-ins-reference-overview.md

```diff
@@ -8,56 +8,38 @@ localization_priority: Priority
 
 # Excel JavaScript API overview
 
-> [!NOTE]
-> TO DO: Kim will be updating the structure and content of this article before the new TOC structure and landing pages are published to the live doc set.
+An Excel add-in interacts with objects in Excel by using the JavaScript API for Office, which includes two JavaScript object models:
 
-You can use the Excel JavaScript API to build add-ins for Excel 2016 or later. The following list shows the high-level Excel objects that are available in the API. Each object page link contains a description of the properties, events, and methods that are available on the object. Explore the links from the menu to learn more.
+* **Excel JavaScript API**: Introduced with Office 2016, the [Excel JavaScript API](/javascript/api/excel) provides strongly-typed objects that you can use to access worksheets, ranges, tables, charts, and more. 
 
-Some of the core Excel objects are listed below for convenience:
+* **Common APIs**: Introduced with Office 2013, the [Common API](/javascript/api/office) can be used to access features such as UI, dialogs, and client settings that are common across multiple types of host applications such as Word, Excel, and PowerPoint.
 
-- [Workbook](/javascript/api/excel/excel.workbook): The top-level object that contains related workbook objects such as worksheets, tables, ranges, etc. It also can be used to list related references.
+This section of the documentation focuses on the Excel JavaScript API, which you'll use to develop the majority of functionality in add-ins that target Excel 2016 or later. For information about the Common API, see [JavaScript API for Office](../javascript-api-for-office.md). 
```

Comment:
Considering Excel 2016 only supports ExcelApi 1.1, maybe we should downplay that. Would it be better for SEO and adoption to say "Excel for Office 365"?